src/test/java/com/bobocode/petros/container/ApplicationAnnotationContainerTest.java

```diff
@@ -4,21 +4,35 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+
 class ApplicationAnnotationContainerTest {
     private ApplicationAnnotationContainer sut;
+    private Map<String, Object> dependencies = new HashMap<>();
 
     @BeforeEach
     void setup(){
         sut = new ApplicationAnnotationContainer("some.package");
+        Class<? extends ApplicationAnnotationContainer> testContainer = sut.getClass();
+        dependencies.put("name",Object.class);
+        try {
+            Field field = testContainer.getDeclaredField("dependencyMap");
+            field.setAccessible(true);
+            field.set(sut, dependencies);
+        } catch (NoSuchFieldException | IllegalAccessException e) {
```

Comment:
> Here we can use for more readable @SneakyThrows

Nope. We avoid using @SneakyThrows in project

Reply:
As i know we might to use only in **test case**.
https://github.com/petros-team/bring/pull/2#discussion_r734954182
@StanislavZabramnyi is it not?

Reply:
While writing tests we definitely shouldn't use try/catch blocks. Feel free to add _throws Exception_ to any test method. @SneakyThrows also won't be an issue. In case if you expect exception - you assert it, if exception is unexpectable then this exception will fail your tests - and that is exactly what we want